docs/dashboards/frequently-asked-questions.md

```diff
@@ -0,0 +1,108 @@
+FAQ
+===
+### How to connect store to the other components
```

Comment:
Can we use numbers for each question?

Reply:
why would we? These are not steps, but each of the points is equally important, etc. Perfect for bullet list :D